EventVisualisation/Base/include/EventVisualisationBase/VisualisationConstants.h

```diff
@@ -0,0 +1,76 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @copyright
+/// © Copyright 2014 Copyright Holders of the ALICE O2 collaboration.
+/// See https://aliceinfo.cern.ch/AliceO2 for details on the Copyright holders.
+/// This software is distributed under the terms of the
+/// GNU General Public License version 3 (GPL Version 3).
+///
+/// License text in a separate file.
+///
+/// In applying this license, CERN does not waive the privileges and immunities
+/// granted to it by virtue of its status as an Intergovernmental Organization
+/// or submit itself to any jurisdiction.
+
+///
+/// \file    VisualisationConstants.h
+/// \author  Jeremi Niedziela
+///
+
+#ifndef ALICE_O2_EVENTVISUALISATION_BASE_VISUALISATIONCONSTANTS_H
+#define ALICE_O2_EVENTVISUALISATION_BASE_VISUALISATIONCONSTANTS_H
+
+#include <string>
+
+namespace o2  {
+namespace EventVisualisation {
+
+enum EDetectors{
+  ACO,
+  EMC,
+  HMP,
+  MCH,
+  PHS,
+  RPH,
+  SDD,
+  SPD,
+  SSD,
+  TOF,
+  TPC,
+  TRD,
+  RND,
+  Ndetectors
+};
+  
+std::string getDetectorName(EDetectors detector)
```

Comment:
`std::string` as return type has some overhead and is probably not necessary here, only constants are returned

Reply:
I also think we have this also elsewhere in the code, why not picking it up from there?

Reply:
Replaced by a const array.